Documentation/AEPEdgeIdentity.md

```diff
@@ -0,0 +1,100 @@
+
+## AndroidX Ads SDK
+Overview: https://developer.android.com/jetpack/androidx/releases/ads#1.0.0-alpha04  
+API reference: https://developer.android.com/reference/androidx/ads/identifier/AdvertisingIdClient  
+Please be advised that based on testing with SDK version 1.0.0-alpha04 on emulator Pixel_3a_API_32_arm64-v8a, the SDK's [`AdvertisingIdClient.isAdvertisingIdProviderAvailable(Context)`](https://developer.android.com/reference/androidx/ads/identifier/AdvertisingIdClient#isAdvertisingIdProviderAvailable(android.content.Context)) does not return `true`, even when a valid app `Context` is provided. See additional source: https://stackoverflow.com/questions/59217195/how-do-i-use-or-implement-an-android-advertising-id-provider  
+Following the guide for ad-id may therefore not work: https://developer.android.com/training/articles/ad-id  
+Note: as of this writing, the last time the AndroidX Ads SDK was updated was January 22, 2020 - Version 1.0.0-alpha04
+
+## Example AndroidX Ads Implementation
+Note that AndroidX Ads SDK doesn't seem to work; `AdvertisingIdClient.isAdvertisingIdProviderAvailable(Context)` does not return `true` even with valid app `Context`
+```kotlin
+if (AdvertisingIdClient.isAdvertisingIdProviderAvailable(context.applicationContext)) {
+    val advertisingIdInfoListenableFuture = AdvertisingIdClient.getAdvertisingIdInfo(context.applicationContext)
+    addCallback(advertisingIdInfoListenableFuture,
+            object : FutureCallback<AdvertisingIdInfo> {
+                override fun onSuccess(adInfo: AdvertisingIdInfo?) {
+                    if (adInfo == null) {
+                        return
+                    }
+                    val id = adInfo.id
+                    val providerPackageName = adInfo.providerPackageName
+                    val isLimitTrackingEnabled = adInfo.isLimitAdTrackingEnabled
+                    Log.d("Custom_Identity_Fragment", "id: $id, providerPackageName: $providerPackageName, isLimitTrackingEnabled: $isLimitTrackingEnabled")
+                }
+
+                override fun onFailure(t: Throwable) {
+                    Log.e("Custom_Identity_Fragment", "Failed to connect to Advertising ID provider: $t")
+                    // Try to connect to the Advertising ID provider again, or fall
+                    // back to an ads solution that doesn't require using the
+                    // Advertising ID library.
+                }
+            },
+            Executors.newSingleThreadExecutor()
+    )
+} else {
+    Log.d("Custom_Identity_Fragment", "The Advertising ID client library is unavailable.")
+    // The Advertising ID client library is unavailable. Use a different
+    // library to perform any required ads use cases.
+}
```

Comment:
Looks like this SDK is not marked as stable yet (currently alpha04), how about we add this section later once it is fully supported?  If you want we can keep a link to the official release page for reference.

Reply:
Ok, that makes sense! I've updated the documentation to note the alpha state and removed the implementation example. I did still keep the testing notes, do you think those should be removed too? I was just thinking it could be helpful to save time in case anyone wanted to try the alpha04 SDK

Reply:
looks good after latest changes, thanks Tim